tests/test_primitives/test_container/test_base_container/test_pickle.py

```diff
@@ -0,0 +1,36 @@
+import pickle  # noqa: S403
+from typing import Any
+
+from hypothesis import example, given
+from hypothesis import strategies as st
+
+from returns.primitives.container import BaseContainer
+
+
+class _CustomClass(object):
+    def __init__(self, inner_value: Any) -> None:
+        self.inner_value = inner_value
+
+    def __eq__(self, other: Any) -> bool:
+        return (
+            type(other) == type(self) and  # noqa: WPS516
+            self.inner_value == other.inner_value
+        )
+
+
+@given(
+    st.one_of(
+        st.integers(),
+        st.floats(allow_nan=False),
+        st.text(),
+        st.booleans(),
+        st.lists(st.text()),
+        st.dictionaries(st.text(), st.integers()),
+    ),
+)
+@example(None)
+@example(_CustomClass(42))
```

Comment:
Why not `st.one_of(..., st.builds(_CustomClass))`? And `st.none()`? https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.none

Reply:
I saw `st.none()` but I wanted to guarantee `None` value will be always present since there's a lot of strategies in the `on_of` already and hypothesis doesn't guarantee it'll draw all the strategies!